package-lock.json

```diff
@@ -7796,13 +7796,22 @@
         "function-bind": "^1.1.1",
         "has": "^1.0.3",
         "has-symbols": "^1.0.1",
-        "is-callable": "^1.2.0",
-        "is-regex": "^1.1.0",
+        "is-callable": "^1.1.5",
```

Comment:
are these changes to package lock intended

Reply:
no, I didn't do anything intentional to change this